assignments/objects.js

```diff
@@ -49,16 +96,44 @@ let example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-let parent = {}
+let parent = {
+  "Name": "Susan",
+  "Age": 70,
+  "speak": function() {
+    return `Hello I am ${this.Name}.`;
+  },
+
+  "child": {
+    "Name": "George",
+    "Age": 50,
+    "speak": function() {
+      return `Hello I am ${this.Name}.`;
+    }
+  },
```

Comment:
Grandchild should be nested on the child object. As written, you have both the child and grandchild objects nested within the parent object.